### PR TITLE
Added DevFest CH 2017

### PR DESCRIPTION
**Pull request by kikoso**

### CODE REVIEW

File: _conferences/devfest-switzerland-2017 copy.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿---
+name: GDG DevFest Switzerland 2017
```
Comment (JakeWharton) [inline _conferences/devfest-switzerland-2017 copy.md]: Remove the city and year from the title

Comment (kikoso) [reply _conferences/devfest-switzerland-2017 copy.md]: Year removed from title. There is no city.

Comment (JakeWharton) [reply _conferences/devfest-switzerland-2017 copy.md]: Country

Comment (kikoso) [reply _conferences/devfest-switzerland-2017 copy.md]: Done

**Comment by JakeWharton** [comment]
Thanks!

**Pull request merged**